aleph_alpha_client/image.py

```diff
@@ -93,6 +96,40 @@ def __init__(
         self.cropping = cropping
         self.controls: Sequence[ImageControl] = controls
 
+    @classmethod
+    def from_image_source(
+        cls,
+        image_source: Union[str, Path, bytes],
+        controls: Optional[Sequence[ImageControl]] = None,
+    ):
+        """
+        Abstraction on top of the existing methods of image initialization.
+        If you are not sure what the exact type of your image, but you know it is either a URL, a file path,
+        or a bytes array, just use the method and we will figure out which of the methods of image initialization to use
+        The image will be [center cropped](https://pytorch.org/vision/stable/transforms.html#torchvision.transforms.CenterCrop)
+        """
+        if isinstance(image_source, Path):
+            return cls.from_file(path=str(image_source), controls=controls)
+
+        if isinstance(image_source, str):

```

Comment:
this should be an elif

Reply:
Alternatively we get rid of the `elif`, `else` below since we always _return_ anyways. We should just use a consistent approach.

Reply:
I am fine with both approaches

Reply:
Added elif